packages/core/src/providers/notifications/model.ts

```diff
@@ -8,12 +8,14 @@ type NotificationPayload = { submittedAt: number } & (
       transaction: TransactionResponse
       receipt: TransactionReceipt
       transactionName?: string
+      originalTransaction?: TransactionResponse
```

Comment:
Please add originalTransaction field to docs so that devs know that when this field is present the transaction was replaced.